.github/workflows/tests.yaml

```
@@ -33,4 +33,4 @@ jobs:
 
             -   uses: "ramsey/composer-install@v1"
 
-            -   run: vendor/bin/paratest ${{ matrix.path }} --colors --verbose
+            -   run: vendor/bin/paratest --processes 3 ${{ matrix.path }} --colors --verbose
```

how about add `--runner=WrapperRunner` ? 

```suggestion
            -   run: vendor/bin/paratest --processes 3 --runner=WrapperRunner ${{ matrix.path }} --colors --verbose
```

---

What does that do?

---

Based on the doc:

```
It spawns one "worker"-process for each parallel process (-p), executes the bootstrapping once and reuses these processes for each test executed
```
https://github.com/paratestphp/paratest#optimizing-speed